SMLHelper/Patchers/CraftDataPatcher.cs

```diff
@@ -75,28 +75,28 @@ internal static void RemoveFromCustomGroup(TechGroup group, TechCategory categor
             if (techGroup == null)
             {
                 // Should never happen, but doesn't hurt to add it.
-                Logger.Log("Invalid TechGroup!");
+                Logger.Log("Invalid TechGroup!", LogLevel.Error);
                 return;
             }
 
             List<TechType> techCategory = techGroup[category];
             if (techCategory == null)
             {
-                Logger.Log($"Invalid TechCategory Combination! TechCategory: {category} TechGroup: {group}");
+                Logger.Log($"Invalid TechCategory Combination! TechCategory: {category} TechGroup: {group}", LogLevel.Error);
                 return;
             }
 
             techCategory.Remove(techType);
 
-            Logger.Log($"Removed \"{techType.AsString():G}\" from groups under \"{group:G}->{category:G}\"");
+            Logger.Log($"Removed \"{techType.AsString():G}\" from groups under \"{group:G}->{category:G}\"", LogLevel.Debug);
         }
 
         internal static void AddToCustomTechData(TechType techType, ITechData techData)
         {
             if (CustomTechData.ContainsKey(techType))
             {
-                Logger.Log($"[ERROR] Custom TechData already exists for '{techType}'. {Environment.NewLine}" +
-                            "All entries will be removed so conflict can be noted and resolved.");
+                Logger.Log($"Custom TechData already exists for '{techType}'. {Environment.NewLine}" +
+                            "All entries will be removed so conflict can be noted and resolved.", LogLevel.Error);
```

Comment:
Let's demote this to be a `Warn` since we should be able to recover from this error.  
Also, this will have to later be reconciled with the changes in #77